addons/core/translations/resources/en-us.yaml

```diff
@@ -97,6 +97,10 @@ host-catalog:
     azure_client_id:
       label: Client ID
       help: The client ID for the client you want Boundary to use.
+    azure_choose_a_provider:
+      description: There are several built-in plugins for the most common providers. You can choose from on of the included plugins or use a custom plugin. Boundary does not yet support rotating credentials.
+    aws_choose_a_provider:
+      description: There are several built-in plugins for the most common providers. You can choose from on of the included plugins or use a custom plugin.
```

Comment:
I am open to suggestions about whether this should live here or not. I did keep the original `choose-a-provider` option because it is still being used by the static host-catalog form.

Reply:
Hmm.  I think it's a good idea to have a default `choose_a_provider`, otherwise these will grow with each provider addition even if they're mostly the same.

Reply:
Agreed with Randy. We could have a default `choose_a_provider` and then if need, extra information coming from adhoc ones.

Reply:
I agree with having a default `choose_a_provider` esp since it is used in `static` forms!

Reply:
I really like the concat approach @calcaide. And yes, I agree that this could grow out of hand as we add more providers. I will remove the AWS specific one and update it to use concatenation when needed.